cmake/modules/FindLibR.cmake

```diff
@@ -186,9 +187,16 @@ execute_process(
   OUTPUT_VARIABLE LIBR_INCLUDE_DIRS
 )
 
+# ask R for the lib dir
+execute_process(
+  COMMAND ${LIBR_EXECUTABLE} "--slave" "--vanilla" "-e" "cat(normalizePath(R.home('lib'), winslash='/'))"
```

Comment:
Rstudio uses the same script: https://github.com/rstudio/rstudio/blob/3e7caf848911476a23e6272a41946d0804a8796b/cmake/modules/FindLibR.cmake#L90-L96

Reply:
oh wow! Ok good, that makes me feel more confident in this 😁